### ISSUE TITLE
Some simple  quetions about val

### ISSUE THREAD

**Issue by jetyang0729** [idstcv/ZenNAS#20]
Hello, @MingLin-home ! I'm studing your zon_ NAS, but I encountered some problems in the process of re engraving. My results are as follows：

python val.py --fp16 --gpu 0 --arch zennet_imagenet1k_flops400M_SE_res224
Evaluate zennet_imagenet1k_flops400M_SE_res224 at 224x224 resolution.
/home/jet/anaconda3/lib/python3.9/site-packages/torchvision/transforms/transforms.py:287: UserWarning: Argument interpolation should be of type InterpolationMode instead of int. Please, use InterpolationMode enum.
  warnings.warn(
---debug use_se in SuperResIDWE1K7(16,40,2,40,1)
---debug use_se in SuperResIDWE1K7(40,64,2,64,1)
---debug use_se in SuperResIDWE4K7(64,96,2,96,5)
---debug use_se in SuperResIDWE2K7(96,224,2,224,5)
loading pretrained parameters...
Using GPU 0.
/home/jet/anaconda3/lib/python3.9/site-packages/torch/nn/functional.py:3631: UserWarning: Default upsampling behavior when mode=bilinear is changed to align_corners=False since 0.4.0. Please specify align_corners=True if the old behavior is desired. See the documentation of nn.Upsample for details.
  warnings.warn(
mini_batch 0, top-1 acc=   0%, top-5 acc=   0%, number of evaluated images=128
mini_batch 100, top-1 acc=   0%, top-5 acc=   0%, number of evaluated images=12928
mini_batch 200, top-1 acc=   0%, top-5 acc=   0%, number of evaluated images=25728
mini_batch 300, top-1 acc=   0%, top-5 acc=1.5625%, number of evaluated images=38528
*** arch=zennet_imagenet1k_flops400M_SE_res224, validation top-1 acc=0.08999999612569809%, top-5 acc=0.343999981880188%, number of evaluated images=50000

1. I hope you can help me to explain the meaning of these results.
2. I want to know : if I want to train a NAS, what procedures will I use?
Thank you very much for your help!

**Comment by MingLin-home** [comment]
Hi jetyang0729,
Thank you for the feedback!
1) You are using the evaluation script. You did not load a pretrained model so the accuracy is nearly zero.
2) To train the model, please check ts_train_image_classification.py for training script. Example can be found in https://github.com/idstcv/ZenNAS/blob/main/scripts/Zen_NAS_ImageNet_latency1.2ms.sh .
